docs/src/modules/components/Demo.js

```diff
@@ -225,6 +241,8 @@ function Demo(props) {
   const handleClickCopy = async () => {
     try {
       await copy(demoData.raw);
+      setSnackBarMessage('The code has been copied!');
```

Comment:
Could you add the strings to the English version of the translations somewhere in this part of the file: https://github.com/mui-org/material-ui/blob/master/docs/translations/translations.json#L69 (The rest of the files will be handled by Crowdin).

(Pedantic aside: I don't think we need the "!", it isn't that exciting. 🙂)